annotation/tests/test_tasks_services.py

```diff
@@ -1,30 +1,240 @@
+        mock_stats
+    )
+    mock_db.query.return_value = mock_query
+    return mock_db
+
+
+@pytest.fixture
+def mock_task():
+    task = MagicMock(ManualAnnotationTask)
+    task.id = 1
+    task.user_id = 2
+    task.job_id = 10
+    task.file_id = 3
+    task.pages = {1, 2, 3}
+    task.is_validation = False
+    yield task
+
+
+@pytest.fixture
+def mock_tenant_data():
+    mock_data = MagicMock(TenantData)
+    mock_data.token = "mock_token"
+    yield mock_data
+
+
+@pytest.fixture
+def response_scores():
+    return [
+        ResponseScore(task_id=2, agreement_score=0.9),
+        ResponseScore(task_id=3, agreement_score=0.7),
+        ResponseScore(task_id=2, agreement_score=0.9),
+    ]
+
+
+@pytest.fixture
+def mock_parse_obj_as():
+    with patch("pydantic.parse_obj_as") as mock:
+        yield mock
+
+
+@pytest.fixture
+def mock_get_unique_scores():
+    with patch(
+        "annotation.tasks.services.get_unique_scores", return_value=None
+    ) as mock:
+        yield mock
+
+
+@pytest.fixture
+def mock_task_metric():
+    with patch("annotation.tasks.services.TaskMetric") as mock:
+        yield mock
+
+
+@pytest.fixture
+def mock_agreement_score_comparing_result():
+    with patch(
+        "annotation.tasks.services.AgreementScoreComparingResult"
+    ) as mock:
+        yield mock
+
+
+@pytest.fixture
+def mock_agreement_score_response():
+    mock_response = [
+        MagicMock(spec=AgreementScoreServiceResponse),
+        MagicMock(spec=AgreementScoreServiceResponse),
+    ]
+    mock_response[0].task_id = 1
+    mock_response[0].agreement_score = [
+        MagicMock(spec=ResponseScore, task_id=2, agreement_score=0.95),
+        MagicMock(spec=ResponseScore, task_id=3, agreement_score=0.8),
+    ]
+    mock_response[1].task_id = 2
+    mock_response[1].agreement_score = [
+        MagicMock(spec=ResponseScore, task_id=1, agreement_score=0.95),
+        MagicMock(spec=ResponseScore, task_id=3, agreement_score=0.85),
+    ]
+    return mock_response
+
+
+@pytest.fixture
+def create_task():
+    def _create_task(status: TaskStatusEnumSchema):
+        task = MagicMock(spec=ManualAnnotationTask)
+        task.status = status
+        task.id = 1
+        return task
+
+    return _create_task
+
+
+@pytest.fixture
+def setup_data():
+    task_data = {
+        "objects": [
+            {"id": 1, "name": "Object A", "value": "Some Value", "links": [2]},
+            {"id": 2, "name": "Object B", "value": "Another Value"},
+        ]
+    }
+
+    all_tasks = {
+        1: [
+            ({"name": "Object A", "value": "Some Value"}, "1"),
+            ({"name": "Object C", "value": "Different Value"}, "3"),
+        ],
+        2: [({"name": "Object B", "value": "Another Value"}, "2")],
+    }
+
+    yield task_data, all_tasks
+
+
+@pytest.fixture
+def mock_session():
+    with patch("annotation.tasks.services.Session", spec=True) as mock_session:
+        yield mock_session()
+
+
+@pytest.fixture
+def mock_dependencies():
```

Comment:
it is not clear at all what `dependencies` are. Please simplify your fixture - it seems like unnecessary.